src/components/Nav/Nav.js

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
 import './Nav.css';
+import { Route } from 'react-router-dom';
 
 const Nav = ({ search, handleSearch, handleDisplayAllMovies }) => {
     return (
     <nav className='nav'>
         <div className='header'>
             <h1>RANCID TOMATILLOS</h1>
         </div>
+        <Route exact path = '/' render={() => 
```

Comment:
This was all that was needed to be able to not render the search input when we click a movie? Great addition.